src/components/alert/alert.css

```diff
@@ -0,0 +1,46 @@
+.alert{
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    margin: 0.5rem;
+    letter-spacing: 1px;
+    border-radius: 0.3rem;
+  }
+  img{
+    height: 16px;
+    width: 16px;
+  }
+  .alert.primary {
+    color:white;
+    background-color: blueviolet;
+    border: 1px solid var(--rb-very-dark-brown);
+  }
+  
+  .alert.error {
+    color:white;
+    background-color: red;
+    border: 1px solid var(--rb-light-black);
+  }
+  
+  .alert.success {
+    color: white;
+    background-color: greenyellow;
+    border: 1px solid var(--rb-very-dark-green);
+  }
+  
+  .alert.warning {
+    color: white;
+    background-color: yellow;
+    border: 1px solid var(--rb-very-dark-brown);
+  }
```

Comment:
be specific as alert class is repeating. try not to repeat classes in CSS

Reply:
sure ,now alert is not repeating